VMS/Shared/LoginDisplay.razor

```diff
@@ -1,19 +1,34 @@
 ﻿<AuthorizeView>
     <Authorized>
-        <li class="nav-item">
-            <a href="" class="text-decoration-none">
-                <span class="material-icons my-notifications" data-icon="notifications_none"></span>
-            </a>
-        </li>
-        <li class="nav-item text-decoration-none">
-            <a href="">
-                <span class="material-icons-outlined my-person" data-icon="account_circle"></span>
+        <li class="user-section nav-item dflex flex-column align-items-center position-relative">
+            <a href="" class="d-block text-decoration-none user-section__link">
```

Comment:
Consider put a temporary url to the href.

Reply:
I created @Routes.User which is equal to "/trang-ca-nhan" for this link tag